src/extension.ts

```diff
@@ -260,6 +289,53 @@ export class Extension implements vscode.Disposable {
     return this.enabled = enabled
   }
 
+  /**
+   * Make all selections in the editor non-empty by selecting at least one character.
+   */
+  normalizeSelections(editor: vscode.TextEditor) {
+    if (this.allowEmptySelections || this.ignoreSelectionChanges)
+      return
+
+    if (this.modeMap.get(editor.document) !== Mode.Normal)
+      return
+
+    // Since this is called every time when selection changes, avoid allocations
+    // unless really needed and iterate manually without using helper functions.
+    let normalizedSelections
```

Comment:
Please add a type annotation here (and explicitly initialize to `undefined`).
```suggestion
    let normalizedSelections: vscode.Selection[] | undefined = undefined
```